### ISSUE TITLE
Clarification for content availability of implicit tileset roots

### ISSUE THREAD

**Issue by javagl** [CesiumGS/3d-tiles#769]
The specification text at https://github.com/CesiumGS/3d-tiles/tree/main/specification/ImplicitTiling#availability-1 says

> If the implicit root tile does not have content then `contentAvailability` shall be omitted.

I was a bit surprised by that statement: It is phrased as a _requirement_ to _omit_ this property, but it only refers to the _root_ tile. I don't see a technical reason for the _requirement_ to omit it _only_ in the _root_ tile, whereas it will be represented by 
``` 
  "contentAvailability" : [ {
    "availableCount" : 0,
    "constant" : 0
  } ],
```
in all other tiles. 

In fact, the validator currently does not check this - and if it did, it would probably make things like the [`SparseImplicitQuadtree`](https://github.com/CesiumGS/3d-tiles-samples/blob/main/1.1/SparseImplicitQuadtree/) invalid, because it does _not_ omit this property. 

An aside: [The schema](https://github.com/CesiumGS/3d-tiles/blob/5015490b8a0ccbdf2e1bae11e9c1514ebd88cbde/specification/schema/Subtree/subtree.schema.json#L67) also does not declare the `contentAvailability` as required. 

It could be clarified that when it is _omitted_, it just means `availableCount:0/constant:0`. But _requiring_ to omit it _only_ in the root tile seems overly constraining, and I don't see a justification for that. I'd lean towards removing this requirement. 

(This came up in the forum at https://community.cesium.com/t/omitting-content-availability-when-using-implicit-tiling/32238 )

**Comment by lilleyse** [comment]
I think this is referring to a tile without a `content` property, e.g.

```json
{
  "root": {
    "boundingVolume": {
      "region": [-1.318, 0.697, -1.319, 0.698, 0, 20]
    },
    "refine": "REPLACE",
    "geometricError": 5000,
    "implicitTiling": {
      "subdivisionScheme": "QUADTREE",
      "availableLevels": 21,
      "subtreeLevels": 7,
      "subtrees": {
        "uri": "subtrees/{level}/{x}/{y}.json"
      }
    }
  }
}
```

`contentAvailability` would be omitted since this implicit tileset contains all empty tiles.

I think the line should be reworded to

> If the implicit root tile does not have a `content` property then `contentAvailability` shall be omitted.

**Comment by javagl** [comment]
Now this makes sense, on the level of the specification. It may raise questions about use-cases for an implicit tileset that does not have content _at all_. But one could make a case for such a tileset serving only as a purely spatial datastructure that stores metadata, for example.

To avoid similar confusion, one could probably consider to elaborate that further, with something like

>If the implicit root tile does not have a `content` property, and therefore does not define a content template URI, then this means that none of the implicit tiles has any content, and the `contentAvailability` in the subtrees shall be omitted.

**Comment by lilleyse** [comment]
@javagl that sounds good. Could you open a PR with that wording?

**Comment by javagl** [comment]
Yes, there are a few "smaller" things that I can try to address in a few small PRs during the weekend, like

- this one
- https://github.com/CesiumGS/3d-tiles/issues/766
- Maybe https://github.com/CesiumGS/3d-tiles/issues/756
